internal/main.go

```diff
@@ -43,11 +43,25 @@ func main() {
 	pflag.BoolVarP(&versionFlag, "version", "V", false, "print the version and exit")
 	pflag.BoolVarP(&options.Strict, "strict", "s", false, "fail on any warning")
 	pflag.BoolVarP(&options.Pretty, "pretty", "p", false, "output formatted json")
-	pflag.StringVar(&input, "input", "", "read from input file instead of stdin")
+	pflag.StringVar(&input, "input", "", "deprecated: read from input file instead of stdin")
+	pflag.Lookup("input").Hidden = true
```

Comment:
Note there's a `MarkDeprecated` we could use instead that hides it and also emits a deprecated message when used: https://github.com/coreos/mantle/pull/1151/commits/d9fb0d258cffcfe136ccc0e15e4c6476c500e754

Reply:
`Deprecated` doesn't itself hide the option; we need both that and `Hidden`.

Reply:
Hmm, weird. That's not what I'm seeing here:

```
{pet ~}$ plume pre-release --help |& grep channel
{pet ~}$ plume pre-release --help |& grep stream
  -S, --stream string             target stream (default "testing")
{pet ~}$ plume pre-release --channel foo --help |& grep stream
Flag --channel has been deprecated, use --stream instead
  -S, --stream string             target stream (default "testing")
```

Reply:
Not sure.  `cobra` may drive `pflags` in a different way.  Anyway, the current arrangement seems to work.